### PR TITLE
Adding listing page

### PR DESCRIPTION
**Pull request by b0llu**
## What to check
- [ ] src/Context/Reducer.context.js
- [ ] src/Pages/LandingPage/LandingComponents/CategoryContainer/CategoryContainer.jsx
- [ ] src/Pages/ListingPage/ListingPage.jsx
- [ ] src/Pages/ListingPage/ListingPageComponents/CategoryBar/CategoryBar.jsx
- [ ] src/Pages/ListingPage/ListingPageComponents/VideoListing/VideoListing.jsx

## What not to check
- [ ] Do Not Check Any Other Files

Live Preview,
https://stormplay.netlify.app/

### CODE REVIEW

File: src/Pages/ListingPage/ListingPageComponents/CategoryBar/CategoryBar.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import "./CategoryBar.css";
+
+export const CategoryBar = ({ category, setCategory }) => {
+  return (
+    <div className="category-bar-container">
+      <ul className="categories">
+        <li
```
Comment (foolhardy21) [inline src/Pages/ListingPage/ListingPageComponents/CategoryBar/CategoryBar.jsx:7]: suggestion - you can `map` over `categories` array here.

**Review by foolhardy21 — APPROVED** [review]
good to go ahead. liked the use of category routes in navigation.

**Review by MayankKumar10 — APPROVED** [review]
Looks Good you can merge it

File: src/Pages/ListingPage/ListingPageComponents/CategoryBar/CategoryBar.jsx (hunk 1 above)
Comment (b0llu) [reply src/Pages/ListingPage/ListingPageComponents/CategoryBar/CategoryBar.jsx:7]: sure will think on that

**Pull request merged**